File: dotnet/DesignPatterns/CreationalDesignPatterns/AbstractFactoryDesignPattern/AbstractFactoryDesignPattern.Example1/Models/Concrete/LinuxButton.cs
using AbstractFactoryDesignPattern.Example1.Models.Abstract;

namespace AbstractFactoryDesignPattern.Example1.Models.Concrete;

public class LinuxButton : Button
{
	public override void Render()
	{
		Console.WriteLine("Rendering Linux Button");
	}
	public override void OnClick()
	{
		Console.WriteLine("Linux Button Clicked");
	}
}

File: java/DesignPatterns/CreationalDesignPatterns/AbstractFactoryDesignPattern/AbstractFactoryDesignPattern.Example1/src/main/java/creational/abstractfactory/example1/models/concretes/LinuxButton.java
package creational.abstractfactory.example1.models.concretes;

import creational.abstractfactory.example1.models.abstracts.Button;

public class LinuxButton extends Button {
    public void render()   { System.out.println("Rendering Linux Button"); }
    public void onClick()  { System.out.println("Linux Button Clicked"); }
}
